addons/editor-delete-button/addon.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "https://raw.githubusercontent.com/ScratchAddons/manifest-schema/dist/schema.json",
+  "name": "Delete button in editor",
+  "description": "Adds a delete button to the file menu that allows you to delete your project.",
+  "credits": [
+    {
+      "name": "Weredime (9gr)",
+      "link": "https://scratch.mit.edu/users/9gr/"
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["projects"]
+    }
+  ],
+  "settings": [
+    {
+      "name": "Reset project before trashing",
+      "type": "boolean",
+      "default": true,
+      "id": "resetProject"
+    }
+  ],
```

Comment:
This setting doesn't work because it doesn't save the project.

Even if it did what it says, I'm not sure if it's useful: all it does is make the project impossible to restore. If you want to keep it, it should be disabled by default because it's dangerous.